src/components/layout.js

```diff
@@ -19,9 +19,9 @@ const Layout = ({ children }) => {
   return (
     <div className={`main-container`}>
       <Header siteTitle={data.site.siteMetadata.title} />
-      <div>
+      <article className={`content`}>
```

Comment:
I think the children might contain many articles, so is this the right markup?

Reply:
this is the section for the main article. There is going to be only one article. Rest of the modules with go in different sections